backend/src/ApiServer/IntegrationTests.fs

```diff
@@ -79,10 +80,14 @@ let loadAndResaveFromTestFile (meta : Meta) : Task<unit> =
   task {
     let oplists =
       let tls = meta |> tryLoadJsonFromDisk LibBackend.Config.Testdata
+      // Save back in the latest format
+      do tls |> Option.tap (saveJsonToDisk LibBackend.Config.Testdata meta)
```

Comment:
To make sure I understand:
- every time we run an integration test, we've been loading them from disk
- this adds another operation where we re-save the oplist to disk, such that:
  - short term, we got rid of the EBinOp usages in the oplists (in a previous commit)
  - long term, we detect other changes similar to how we detect serialization changes with Vanilla
  - 
?

Reply:
Yes